### PR TITLE
Implement Frank's mind map

### PR DESCRIPTION
**Pull request by onyb**
https://www.mindmup.com/#m:a150006750edb50132dfcf1222860cd564

### CODE REVIEW

File: station_data.py [hunk 1]
```diff
@@ -62,45 +63,52 @@ def get_events(station, start, end):
...
 
-        event['timestamp'] = int(row[2])
-        event['nanoseconds'] = int(row[3])
+        event['location'] = {
+                             'latitude'  : station_data['latitude'],
+                             'longitude' : station_data['longitude'],
+                             'altitude'  : station_data['altitude']
+                            }
+
+        event['time'] = row[0] + ' ' + row[1] + '%9d'%(int(row[2]))
```
Comment (153957) [inline station_data.py]: I would add a dot between the number of seconds and nanoseconds. (to indicate the nanoseconds are fractional seconds)

Comment (onyb) [inline station_data.py]: @153957: Thanks for pointing out. I have corrected this.

**Comment by RaoOfPhysics** [comment]
DST is a social construct, which doesn't change the position of the sun. So, that's tricky. I would think that the times should always be the non-DST ones. @berghaus to comment?

**Comment by 153957** [comment]
Since we are looking at cosmic rays (mostly not from the Sun) would GMST (or LST) then not be more appropriate than local time?

**Comment by onyb** [comment]
@153957: I let it remain as local time. The program parsing this JSON can obviously convert the timestamp to GMT or any other format.

So far Daylight Savings Time is concerned, I am looking into it right now.

**Comment by onyb** [comment]
@153957, @RaoOfPhysics: Daylight Savings Time changes is already handled by the API that I have used.

**Comment by 153957** [comment]
Handled how? You only give it the location so I assume it returns the current DST, not the DST at time of the event..?

**Comment by onyb** [comment]
@153957: I am afraid you are right. This needs more work than I thought.

**Comment by 153957** [comment]
The main reason we do not use a local time (or UTC) is because of DST (and leap seconds). Date times are not unique in those cases. This increases the probability for errors when determining event rates or combining it with other data sources (for instance weather data). It is easiest to keep it as GPS (or UTC) time, prevents some headaches.. (we use GPS time with the UNIX epoch)

**Pull request merged**